services/workflows-service/src/workflow/workflow.controller.external.ts

```diff
@@ -34,9 +38,11 @@ import { WorkflowDefinitionWhereUniqueInput } from './dtos/workflow-where-unique
 import { RunnableWorkflowData } from './types';
 import { WorkflowDefinitionModel } from './workflow-definition.model';
 import { WorkflowService } from './workflow.service';
+import { TWorkflowExtenstion } from './schemas/extenstions.schemas';
 
+export const WORKGLOW_TAG = 'Workflows';
 @swagger.ApiBearerAuth()
-@swagger.ApiTags('Workflows')
+@swagger.ApiTags(WORKGLOW_TAG)
```

Comment:
If I remember correctly, there are mixed endpoints in this controller, some related to Workflow Definition, some to Workflow Runtime... This change could create confusion